packages/api/src/hooks/middleware/applyMiddleware.js

```diff
@@ -1,9 +1,9 @@
 import React, { isValidElement } from 'react';
-
-import concatMiddleware from './concatMiddleware';
 import ErrorBox from '../internal/ErrorBox';
+import concatMiddleware from './concatMiddleware';
```

Comment:
Sorting order?

At some points, we should learn/fix, if we should continue to separately group imports from `node_modules` or local, vs. all imports in a single group.